forge-gui/res/cardsfolder/upcoming/the_five_doctors.txt

```diff
@@ -0,0 +1,8 @@
+Name:The Five Doctors
+ManaCost:5 G
+Types:Sorcery
+K:Kicker:5
+A:SP$ ChangeZone | Origin$ Library | OriginAlternative$ Graveyard | AlternativeMessage$ Would you like to search your library with this ability? If you do, your library will be shuffled. | Destination$ Hand | DestinationAlternative$ Battlefield | ChangeType$ Doctor | ChangeNum$ 5 | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ0 | SubAbility$ DBChangeZone | SpellDescription$ Search your library and/or graveyard for up to five Doctor cards, reveal them, and put them into your hand. If you search your library this way, shuffle. If this spell was kicked, put those cards onto the battlefield instead of putting them into your hand.
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Doctor | ChangeNum$ 5 | Condition$ Kicked

```

Comment:
The way this is scripted you would only be able to search the graveyard if the spell was not kicked. The Condition stuff in line 5 means the line would be completely skipped. Then line 6 has none of the alternative params.

IMO ideally it would use one ChangeZone line but that might require tweaks to ChangeZoneEffect

If you want to work on other things I could hammer this out.

Reply:
Sorting this out in #4228